test/entity_test.rb

```diff
@@ -201,4 +202,26 @@ class Camera
       end
     end
   end
+
+  describe 'dirty tracking' do
+
+    it 'dirty state when created' do
+      book = Book.new(title: 'Crime and Punishment', author: 'F. Dostoevskiy')
+      book.changed?.must_equal true
+    end
+
+    it 'changes dirty state' do
+      book1 = Book.new
+      book1.changed?.must_equal false
+      book1.title = 'War and Peace'
+      book1.changed?.must_equal true
+      book1.changed_attributes.must_include :title
+    end
+
```

Comment:
Can you please add another test for `#update`? We expect it to already work because it uses internally `public_send("#{attribute}=", value)`. But if we change that implementation, dirty tracking won't work anymore. We need a test to prevent regressions.